src/sagemaker/remote_function/runtime_environment/runtime_environment_manager.py

```diff
@@ -368,8 +367,8 @@ def _validate_sagemaker_pysdk_version(self, client_sagemaker_pysdk_version):
                 "Inconsistent sagemaker versions found: "
                 "sagemaker pysdk version found in the container is "
                 "'%s' which does not match the '%s' on the local client. "
-                "Please make sure that the python version used in the training container "
-                "is the same as the local python version in case of unexpected behaviors.",
+                "Please make sure that the pysdk version used in the training container "
```

Comment:
nit: clarify sagemaker python SDK? pysdk is very vague

Reply:
Let's official terminologies in our docs and error messages